src/transactions/selectors.ts

```diff
@@ -118,3 +118,11 @@ export const pendingStandbyTxHashesByNetworkIdSelector = createSelector(
 
 const feedFirstPage = (state: RootState) => state.transactions.feedFirstPage
 export const feedFirstPageSelector = createSelector(feedFirstPage, (feed) => feed)
+
+const hasNewUnknownCompletedTransactions = (state: RootState) =>
+  state.transactions.hasNewUnknownCompletedTransactions
+
+export const hasNewUnknownCompletedTransactionsSelector = createSelector(
```

Comment:
Why do we need this selector? This looks like it's just directly passing the result from `hasNewUnknownCompletedTransactions` -- could we just export and use that?

Reply:
@jophish Yes, will do!

Reply:
@jophish Sometimes when I retrieve data with the selector that is not using `createSelector`  - it throws the `You must pass a selector to useSelector` error on tests. This was also the case this time. It seems to work now though after the following changes:
- removed `createSelector` hook and got `You must pass a selector to useSelector` error again
- added `createSelector` back - error disappeared
- removed `createSelector` again - now it's all good and no errors 🤷 

I really have no idea why is this happening. I've also experienced this exact behaviour when using `allStandByTransactions` selector a few weeks ago and the issue was resolved in the exact same way. It just took a few more back-and-forth tries with removing/adding back the `createSelector` piece but if there's something obvious that I'm missing - please, let me know!